packages/mds-service-helpers/server/index.ts

```diff
@@ -32,18 +35,46 @@ const ServiceMonitor = async <TServiceInterface>(
   service: ServiceProvider<TServiceInterface>,
   options: ProcessMonitorOptions = {}
 ): Promise<ProcessMonitor> => {
-  const { interval = hours(1), signals = ['SIGINT', 'SIGTERM'] } = options
+  const {
+    interval = hours(1),
+    signals = ['SIGINT', 'SIGTERM'],
+    retries = 15,
+    minTimeout = seconds(2),
+    maxTimeout = minutes(2),
+    ...retryOptions
+  } = options
 
   const {
     env: { npm_package_name, npm_package_version, npm_package_git_commit }
   } = process
 
   const version = `${npm_package_name} v${npm_package_version} (${npm_package_git_commit ?? 'local'})`
 
-  logger.info(`Initializing service ${version}`)
-
   // Initialize the service
-  await service.initialize()
+  try {
+    await retry(
```

Comment:
I know using `retry` enables us to have some better logging, but shouldn't we just lean on TypeORMs retry model?

Reply:
@wesleyfsmith This could be an interesting package to add to the sim